packages/mermaid/src/rendering-util/rendering-elements/edges.js

```diff
@@ -606,7 +573,6 @@ export const insertEdge = function (elem, edge, clusterDb, diagramType, startNod
 
     svgPath = select(svgPathNode)
       .select('path')
-      // .attr('d', lineFunction(lineData))
```

Comment:
Leave the lineFunction for now, we can remove it in the  big PR. 
Still working out some issue around it.